content/posts/irods-development-update-april-2023.md

```diff
@@ -0,0 +1,222 @@
+Title: iRODS Development Update: April 2023
+Date: 2023-04-20 12:00
+Author: Kory Draughn
+Slug: irods-development-update-april-2023
+Status: published
+
+
+Happy Thursday Everyone,
+
+A lot has happened since last month. We are closer than ever to a release of iRODS 4.2.12. Our annual [iRODS User Group Meeting (UGM)](https://irods.org/ugm2023) is right around the corner as well. Registration is open for people who want to learn more about iRODS.
+
+For the server, we've made it so that `iexit` no longer removes the authentication file for the service account. This change protects administrators from losing the ability to manage their server. Of course, if you really need to remove the authentication file, then you have to option to forcibly remove it. `imkdir` has been tweaked to improve the performance in regard to inheritance and large collections. We've also implemented new microservices which give users the ability to inspect JSON objects using the iRODS Rule Language. This is specifically useful for handling PEPs that deal with JSON strings.
+
+We've fixed some important issues in the Logical Quotas rule engine plugin. It now uses the local rodsadmin user to update quota values. This avoids potential permission issues. Not only that, but it no longer crashes the agent when given unsupported rule text via `irule`.
+
+Development on the new iRODS CLI has started again. We don't have any specifics at this time, but as soon as we have something to show, you'll know about it.
+
+The Testing Environment has received a nice improvement in regard to topology testing. Topology tests can now be run in parallel. Before this change, the tests took upwards of 8 hours to complete. With this latest change, we've witnessed the tests taking as little as 2.25 hours to complete.
```

Comment:
"2.25 hours to complete" -> "2.25 hours to complete on our development machines"

Reply:
Done.